index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div class="hamburger">
+                <span class="bars hamburger-bars"></span>
+                <span class="bars hamburger-bars"></span>
+                <span class="bars hamburger-bars"></span>
+            </div>
+            <ul class="menu">
+              <li><a href="#">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Speaker</a></li>
+              <li><a href="#">Sponsors</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Register Now</a></li>
+           </ul>
+        </nav>
+    </header>
+    <section class="hero_section">
+
+        <div class="content1">
+         <h2>"Register Now"</h2>
+         <h1>UNLOCK THE </br> POWER OF DATA </br> SCIENCE WITH AI</h1>
```

Comment:
This text <h>UNLOCK THE
POWER OF DATA
SCIENCE WITH AI</h1> should use the font family of Lato according to the design. Please fix this.